traitsui/qt4/tabular_editor.py

```diff
@@ -433,6 +438,14 @@ def _scroll_to_row_changed(self, row):
             self.factory.scroll_to_row_hint, self.control.PositionAtCenter)
         self.control.scrollTo(self.model.index(row, 0), scroll_hint)
 
+    def _scroll_to_column_changed(self, column):
+        """ Scroll to the given column.
+        """
+        import ipdb; ipdb.set_trace()
+        scroll_hint = self.scroll_to_row_hint_map.get(
+            self.factory.scroll_to_row_hint, self.control.PositionAtCenter)
+        self.control.scrollTo(self.model.index(0, column), scroll_hint)
```

Comment:
I think this will always scroll back up to the top (the row index is 0).  You might use `activated_row` instead?

Reply:
Thanks Corran, made the change. I noticed that scroll_to_row has this issue, and that's what I am poking around to see if I can store a 'selected_column' property as well.

Reply:
Don't worry about this in this PR - you'd want an `activated_row` property rather than a `selected_row` because selections can potentially span multiple rows (and/or columns).

If you want to handle this, then you can make another PR, but I don't think it's urgent.